includes/definitions/gaia.yaml

```diff
@@ -13,7 +13,9 @@ discovery:
         sysObjectID:
             - .1.3.6.1.4.1.2620.1.6.123.1
     -
-        sysObjectID: .1.3.6.1.4.1.8072.3.2.10
+        sysObjectID: 
+            - .1.3.6.1.4.1.8072.3.2.10
+            - .1.3.6.1.4.1.2620.1.1
```

Comment:
This is in the wrong place, move it up a few lines.
If you put it here it requires an additional snmp get for those devices, when we already know they are checkpoint by the sysObjectID.